source/guide_mailman.rst

```diff
@@ -0,0 +1,235 @@
+
+Step 4
+------
+
+If you want the webinterface to be available publically, we need to create a couple of SymLinks and an htaccess-file.
+
+::
+
+ [isabell@stardust ~]$ cd /var/www/virtual/isabell/html
+ [isabell@stardust ~]$ ln -s /var/www/virtual/isabell/mailman/cgi-bin ./mailman
+ [isabell@stardust ~]$ ln -s /var/www/virtual/isabell/mailman/archives/public ./pipermail
+
+Create the file ``/var/www/virtual/isabell/mailman/cgi-bin/.htaccess`` with the following content:
+
+::
+
+ Options +ExecCGI
+ SetHandler cgi-script
+
+Finally, we need to adjust file permissions for the Mailman_ cgi-scripts to run:
+
+::
+
+ [isabell@stardust ~]$ chmod -R 0755 /var/www/virtual/isabell/mailman/cgi-bin
+
+Step 5
+------
+
+Because Mailman_ doesn't handle our .qmail-configuration automatically, we need to help it create the necessary aliases. The following script is based on the script provided in the official installation instructions and may be used by placing it in your Uberspace home-folder:
+
+::
+
+ #!/bin/sh
+ if [ $# = 1 ]; then
+ i=$1
+ echo Making links to $i in the current directory...
+ echo "|preline /var/www/virtual/`whoami`/mailman/mail/mailman post $i" > .qmail-$i
+ echo "|preline /var/www/virtual/`whoami`/mailman/mail/mailman admin $i" > .qmail-$i-admin
+ echo "|preline /var/www/virtual/`whoami`/mailman/mail/mailman bounces $i" > .qmail-$i-bounces
+ echo "|preline /var/www/virtual/`whoami`/mailman/mail/mailman confirm $i" > .qmail-$i-confirm
+ echo "|preline /var/www/virtual/`whoami`/mailman/mail/mailman join $i" > .qmail-$i-join
+ echo "|preline /var/www/virtual/`whoami`/mailman/mail/mailman leave $i" > .qmail-$i-leave
+ echo "|preline /var/www/virtual/`whoami`/mailman/mail/mailman owner $i" > .qmail-$i-owner
+ echo "|preline /var/www/virtual/`whoami`/mailman/mail/mailman request $i" > .qmail-$i-request
+ echo "|preline /var/www/virtual/`whoami`/mailman/mail/mailman subscribe $i" > .qmail-$i-subscribe
+ echo "|preline /var/www/virtual/`whoami`/mailman/mail/mailman unsubscribe $i" > .qmail-$i-unsubscribe
+ fi
+
+After creating a list via the webinterface, you can then run this script to create the required .qmail-files (like ``~/addlist.sh listname`` if you stored it as ``addlist.sh`` and want to create aliases for a list ``listname``).
+
+Configuration
+=============
+
+By now we have installed all the necessary files - let's tell them what they should actually do. In ``/var/www/virtual/isabell/mailman/Mailman``, you will find two important files for configuration: ``Defaults.py`` and ``mm_cfg.py``. In the former, you will be able to review every settings default value. In the latter, you can change the value of each setting.
+
+For our Uberspace, we should set ``DEFAULT_URL_HOST`` and ``DEFAULT_EMAIL_HOST`` to the name of your Uberspace host (i.e. ``stardust.uberspace.de``).
+
+You will also need to create a mailbox_ for Mailman to use and set the following options respectively:
+
+::
+
+ SMTP_AUTH = True
```

Comment:
Again, please specify exactly which file should be changed and – if applicable – that there are already default / dummy values that need to be replaced.